app/scripts/lib/transaction/institutional-snap/deferred-publication-hooks.ts

```diff
@@ -0,0 +1,186 @@
+import {
+  TransactionController,
+  TransactionEnvelopeType,
+  TransactionMeta,
+  TransactionStatus,
+} from '@metamask/transaction-controller';
+import type { HandleSnapRequest } from '@metamask/snaps-controllers';
+import { SnapId } from '@metamask/snaps-sdk';
+import { HandlerType } from '@metamask/snaps-utils';
+import { BaseController, RestrictedMessenger } from '@metamask/base-controller';
+import InstitutionalWalletSnap from '@metamask/institutional-wallet-snap/dist/preinstalled-snap.json';
+import { AccountsControllerGetAccountByAddressAction } from '@metamask/accounts-controller';
+
+const snapId = InstitutionalWalletSnap.snapId as SnapId;
```

Comment:
Minor, capitals and underscores for constants?

Reply:
Done - changed to SNAP_ID https://github.com/MetaMask/metamask-extension/pull/30180/files#diff-fc1652c483153bd55fe46a633567c33518fd9310f5ad683244926d8e8e100f04R14